Apromore-Core-Components/Apromore-BPMNEditor/src/main/scripts/editor.js

```diff
@@ -56,8 +56,32 @@ Apromore.Editor = {
         return Ext.get(this.rootNode).parent("div{overflow=auto}", true);
     },
 
+    updateUndoRedo: function () {
+        try {
+            var undo = jq("button[title='Undo the last action']");
+            var redo = jq("button[title='Redo the last undone action']");
+            if (this.canUndo()) {
+                undo.removeClass('disabled');
+            } else {
+                undo.addClass('disabled');
+            }
+            if (this.canRedo()) {
+                redo.removeClass('disabled');
+            } else {
+                redo.addClass('disabled');
+            }
+        } catch(e) {
+            // pass
```

Comment:
Can we log or even show a user-friendly message here (e.g. "Unexpected error occurred when update button status, Error message: ..."? This is to at least provide a clue to support and fix bugs.